src/main/java/com/vinylteam/vinyl/service/PasswordRecoveryService.java

```diff
@@ -1,10 +1,10 @@
 package com.vinylteam.vinyl.service;
 
-import com.vinylteam.vinyl.web.dto.UserChangeProfileInfoRequest;
+import com.vinylteam.vinyl.web.dto.UserInfoRequest;
 
 public interface PasswordRecoveryService {
 
-    void changePassword(UserChangeProfileInfoRequest userProfileInfo);
+    void changePassword(UserInfoRequest userProfileInfo);
```

Comment:
Are you sure it must be "userProfileInfo" not "userInfoRequest" or just "request" ?

Reply:
Could you please fix this name also in the interface ?